plugins/azure_sentinel/plugin.spec.yaml

```diff
@@ -0,0 +1,879 @@
+    output:
+      id:
+        title: Full Incident ID
+        description: Full incident ID
+        type: string
+      name:
+        title: Incident ID
+        description: Incident name - short id
+        type: string
+      type:
+        title: type
+        type: string
+      etag:
+        title: etag
+        type: string
+      properties:
+        title: Incident properties
+        description: Incident properties object.
+        type: IncidentProperties
+  list_incidents:
+    title: List incidents
+    description: Gets all the incidents 
+    input:
+      resourceGroupName:
+        title: Resource Group Name
+        required: true
+        description: The name of the resource group within the user's subscription. The name is case insensitive.
+        type: string
+      subscriptionId:
+        title: Subscription ID
+        required: true
+        description: "Azure subscription ID Regex pattern: ^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$"
+        type: string
+      workspaceName:
+        title: Workspace Name
+        required: true
+        description: The name of the workspace
+        type: string
+      api-version:
+        title: API version
+        description: The API version to use for this operation.
+        required: true
+        default: 2021-04-01
+        type: string
+      filter:
+        title: filter the results
+        required: false
+        description: Filters the results, based on a Boolean condition. Optional.
+        type: string
+      orderBy:
+        title: order the results
+        required: false
+        description: sorts the results 
+        type: string
+      top:
+        title: Return top results
+        required: false
+        description: Return only n first results. 
+        type: integer
+    output:
+      incidents:
+        title: List of incidents
+        description: list of incidents objects 
+        type: "[]Incident"
+  get_incident:
+    title: Get Incident
+    description: Get a specific incident from a given workspace and resource group
+    input:
+      incidentId:
+        title: Incident ID
+        required: true
+        description: Incident ID
+        type: string
+      resourceGroupName:
+        title: Resource Group Name
+        required: true
+        description: The name of the resource group within the user's subscription. The name is case insensitive.
+        type: string
+      subscriptionId:
+        title: Subscription ID
+        required: true
+        description: "Azure subscription ID Regex pattern: ^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$"
+        type: string
+      workspaceName:
+        title: Workspace Name
+        required: true
+        description: The name of the workspace
+        type: string
+      api-version:
+        title: API version
+        description: The API version to use for this operation.
+        required: true
+        default: 2021-04-01
+        type: string
```

Comment:
I don't think user should use that input. Maybe that should be hardcoded somewhere in the code?

Reply:
You are right! will do